src/HotChocolate/Skimmed/src/Skimmed/Serialization/SchemaParser.cs

```diff
@@ -634,9 +644,22 @@ private static void BuildDirectiveCollection(
                 directiveNode.Name.Value,
                 out var directiveType))
             {
-                directiveType = new DirectiveDefinition(directiveNode.Name.Value);
-                // TODO: This is problematic
-                directiveType.IsRepeatable = true;
+                if (directiveNode.Name.Value == BuiltIns.Deprecated.Name)
+                {
+                    directiveType = BuiltIns.Deprecated.Create(schema);
+                }
+                else if (directiveNode.Name.Value == BuiltIns.SpecifiedBy.Name)
+                {
+                    directiveType = BuiltIns.SpecifiedBy.Create(schema);
+                }
+                else
+                {
+                    directiveType = new DirectiveDefinition(directiveNode.Name.Value);
+                    // TODO: This is problematic, but currently necessary for the Fusion
+                    // directives to work, since they don't have definitions in the source schema.
+                    directiveType.IsRepeatable = true;
```

Comment:
so you make directives repeatable... this should rather be a composition error I think.

Reply:
Ah, I see... we stub them ... yes ... we might should stub the more properly like with types .... I will think on it a bit

Reply:
Yes, this isn't something I added, it was already there. I tried to remove it but, but decided to just add a comment, since I felt it was out of scope.